Resources/config/extractor_relations.yml

```diff
@@ -1,6 +1,5 @@
 parameters:
     # Sql relations classes
-    ongr_connections.sql_relations.simple_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SimpleSqlRelation
-    ongr_connections.sql_relations.composed_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\ComposedSqlRelation
+    ongr_connections.sql_relations.sql_relation.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SqlRelation
```

Comment:
After a bit of discussion we (@GrandLTU and myself) have decided that the most fitting name would be `extraction_descriptor`, since it is a descriptive class used by `extractor` to extract relevant data (an `extraction`). What used to be an `relation collection` would subsequently become `extraction collection` or simply `extractions`. What is your opinion about this, @trylika?